bots/psuteam7bot/crusader.js

```diff
@@ -66,6 +66,71 @@ crusader.takeAttackerAction = (self) => {
 
     const visibleRobots = self.getVisibleRobots();
     const attackable = combat.filterByAttackable(self, visibleRobots);
+    const unattackable = combat.filterByUnattackable(self, visibleRobots);
+
+    // TODO Crusader-vs-Prophet micro, single prophet, 1 enemy prophet in attackable range
+    if((attackable.length === 1 && attackable[0].unit === 4) && movement.getDistance(self.me, attackable[0]) >= SPECS.UNITS[attackable[0].unit].ATTACK_RADIUS[0])
```

Comment:
Should this be `movement.getDistance(self.me, attackable[0]) > SPECS.UNITS[attackable[0].unit].ATTACK_RADIUS[0])` and not `>=` ?

Reply:
never mind figured it out its all good